servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/HttpAttributesGetter.java

```diff
@@ -96,6 +104,37 @@ public final String getNetworkProtocolVersion(
         return response.version().fullVersion();
     }
 
+    @Nullable
+    @Override
+    public String getNetworkTransport(RequestInfo requestInfo, @Nullable HttpResponseMetaData responseMetaData) {
+        ConnectionInfo connectionInfo = requestInfo.connectionInfo();
+        if (connectionInfo == null) {
+            return null;
+        }
+        if (connectionInfo.remoteAddress() instanceof InetSocketAddress) {
+            return TCP;
+        } else if (connectionInfo.remoteAddress() instanceof DomainSocketAddress) {
+            return UNIX;
+        } else {
+            // we don't know.
+            return null;
+        }
+    }
+
+    @Nullable
+    @Override
+    public String getNetworkType(RequestInfo requestInfo, @Nullable HttpResponseMetaData responseMetaData) {
+        ConnectionInfo connectionInfo = requestInfo.connectionInfo();
+        if (connectionInfo == null) {
+            return null;
+        }
+        if (connectionInfo.remoteAddress() instanceof InetSocketAddress) {
+            return ((InetSocketAddress) connectionInfo.remoteAddress()).getAddress() instanceof Inet6Address ?
+                    IPV6 : IPV4;
```

Comment:
Consider also checking for `Inet4Address` instead of using unchecked "else" path. Who knows if there might be a 3rd variant